message_sender/factory.py

```diff
@@ -339,6 +344,7 @@ def send_hsm(self, whatsapp_id, content):
             urllib_parse.urljoin(self.api_url, "/v1/messages"),
             json={
                 "to": whatsapp_id,
+                "ttl": self.ttl,
```

Comment:
I'm not sure what will happen if we send `{..., "ttl": null, ..}` to the whatsapp API, have we tried that to make sure that it's supported? Otherwise, I think we should only add the `ttl` field to the request when it's been set, since it's an optional config parameter.

Reply:
from the docs, it is not a required field so I'm assuming having a `None` value is possible. 

How do I only add it to a request? is it done outside this method?

Reply:
You can define the data in a variable:
```data = {
...
}```
And then add it if it exists:
```
If ttl is not None:
    data["ttl"] = ttl
```